Formula/c/cpp-lazy.rb

```diff
@@ -0,0 +1,38 @@
+class CppLazy < Formula
+  desc "C++11 (and onwards) library for lazy evaluation"
+  homepage "https://github.com/Kaaserne/cpp-lazy"
+  url "https://github.com/Kaaserne/cpp-lazy/archive/refs/tags/v8.0.0.tar.gz"
+  sha256 "02f29ff1e6b44fc0e34cb7c651ee4b4f806c63dfc699fc135db794424fdc119f"
+  license "MIT"
+  head "https://github.com/Kaaserne/cpp-lazy.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "fmt" => :test
+
+  def install
+    args = %w[
+      -DCPP-LAZY_USE_STANDALONE=ON
+    ]
+
+    system "cmake", "-S", ".", "-B", "builddir", *args, *std_cmake_args
+    system "cmake", "--build", "builddir"
+    system "cmake", "--install", "builddir"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~CPP
+      #include <cpp-lazy-7.0.2/Lz/Map.hpp>
```

Comment:
Why 7.0.2?

Reply:
I don't know how to solve this, may be add a link by oneself

Reply:
Is there a pull request or issue upstream to resolve this?

Reply:
not yet

Reply:
https://github.com/Kaaserne/cpp-lazy/issues/167#issuecomment-2733508859